LuaMenu/widgets/gui_campaign_handler.lua

```diff
@@ -731,14 +731,26 @@ local function SelectPlanet(popupOverlay, planetHandler, planetID, planetData, s
 		children = fluffLabels,
 	}
 
-	local planetDesc = TextBox:New {
-		x = 8,
-		y = "30%",
-		right = 4,
-		bottom = "25%",
-		text = planetData.infoDisplay.text,
```

Comment:
How about just changin this single line to `text = (startable or Configuration.debugMode) and planetData.infoDisplay.text or "This planet will need to be approached for further study.",`

Reply:
I assume this is what you mean? Seemed the cleanest way to implement